CONTRIBUTING.rst

```diff
@@ -51,11 +51,33 @@ If you are unsure what that means, here is a set-up workflow you may wish to fol
 
      git push -u origin topic_of_your_contribution
 
-
 (If any of the above seems overwhelming, you can look up the `Git documentation
 <http://git-scm.com/documentation>`_ on the web.)
 
 
+Releases and Changelog
+----------------------
+
+Heudiconv uses the `auto <https://intuit.github.io/auto/>`_ tool to generate the changelog and automatically release the project.
```

Comment:
Let's use consistent project name unless referring to command line utility `heudiconv` :

```suggestion
HeuDiConv uses the `auto <https://intuit.github.io/auto/>`_ tool to generate the changelog and automatically release the project.
```